lib/miq_expression.rb

```diff
@@ -625,10 +625,14 @@ def sql_supports_atom?(exp)
       # => TODO: support count of child relationship
       return false if exp[operator].key?("count")
 
-      return field_in_sql?(exp[operator]["field"])
+      return field_in_sql?(exp[operator]["field"]) && value_in_sql?(exp[operator]["value"])
     end
   end
 
+  def value_in_sql?(value)
+    !Field.valid_field?(value) || Field.parse(value).attribute_supported_by_sql?
```

Comment:
```sql
field = value
```

The value is valid if it is a static value, OR if it is a value that can be represented as a sql friendly sql statement.